src/main/java/io/jenkins/plugins/pipeline_cloudwatch_logs/LogStreamState.java

```diff
@@ -449,30 +435,31 @@ private void process() {
             }
             // TODO as per https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_PutLogEvents.html verify that total size <1Mb (no documented error class for excess size?)
             assert !processing.isEmpty();
-            processing.sort(Comparator.comparing(InputLogEvent::getTimestamp));
+            processing.sort(Comparator.comparing(InputLogEvent::timestamp));
             while (true) {
                 try {
-                    PutLogEventsResult result = currentClient.putLogEvents(new PutLogEventsRequest().
-                            withLogGroupName(logGroupName).
-                            withLogStreamName(logStreamName).
-                            withSequenceToken(sequenceToken).
-                            withLogEvents(processing));
-                    sequenceToken = result.getNextSequenceToken();
-                    RejectedLogEventsInfo problems = result.getRejectedLogEventsInfo();
+                    var _sequenceToken = sequenceToken;
```

Comment:
Irritatingly, Java lambdas refuse to let you refer to= a variable which is not effectively final, despite the fact that 95% of the time what you meant was to bind the current value when creating the lambda.